commit d52e8fceb89f6a2a1fb7f8b8b059ce774193e3df
Author: David Hummel <6109326+hummeltech@users.noreply.github.com>
Date:   Mon Aug 9 11:54:43 2021 -0700

    Resolve PHPUnit warnings
    
    `assertRegExp() is deprecated and will be removed in PHPUnit 10.
    Refactor your code to use assertMatchesRegularExpression() instead.`

diff --git a/test/php/Nominatim/DBTest.php b/test/php/Nominatim/DBTest.php
index 1a2ecc86..cb18ae32 100644
--- a/test/php/Nominatim/DBTest.php
+++ b/test/php/Nominatim/DBTest.php
@@ -132,7 +132,7 @@ class DBTest extends \PHPUnit\Framework\TestCase
                             getenv('UNIT_TEST_DSN') :
                             'pgsql:dbname=nominatim_unit_tests';
 
-        $this->assertRegExp(
+        $this->assertMatchesRegularExpression(
             '/unit_test/',
             $unit_test_dsn,
             'Test database will get destroyed, thus should have a name like unit_test to be safe'
diff --git a/test/php/Nominatim/DatabaseErrorTest.php b/test/php/Nominatim/DatabaseErrorTest.php
index da156493..339d1cc1 100644
--- a/test/php/Nominatim/DatabaseErrorTest.php
+++ b/test/php/Nominatim/DatabaseErrorTest.php
@@ -26,6 +26,6 @@ class DatabaseErrorTest extends \PHPUnit\Framework\TestCase
     public function testSqlObjectDump()
     {
         $oErr = new DatabaseError('Sql error', 123, null, array('one' => 'two'));
-        $this->assertRegExp('/two/', $oErr->getSqlDebugDump());
+        $this->assertMatchesRegularExpression('/two/', $oErr->getSqlDebugDump());
     }
 }
